extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigationOnSiteNotifications_index.php

```diff
@@ -0,0 +1,19 @@
+<div id="onSiteNotificationsEntryPoint"
+     class="wds-global-navigation__notifications-menu wds-global-navigation__notifications-menu wds-dropdown notifications-container">
```

Comment:
`wds-global-navigation__notifications-menu` is duplicated.
What is `notifications-container` for? I don't think it's necessary.